From d2ca9bb8304d1f126f6fe6351bf921880d24ba50 Mon Sep 17 00:00:00 2001
From: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Date: Tue, 20 Feb 2024 14:37:40 +0100
Subject: [PATCH 080/120] match-stm32: remove useless STM32_RESET config from
 Kconfig

CONFIG_RESET_STM32_xxx config are selected automatically in
drivers/reset/stm32/Kconfig

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/u-boot.git 07067c27500e6e3294b137dfdb1910577bd91770 match-stm32: remove useless STM32_RESET config from Kconfig]
Signed-off-by: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Change-Id: I561016ce7ce1b79f966c8b6f66fa0b57800e3eab
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/u-boot/+/361429
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Patrice CHOTARD <patrice.chotard@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Patrice CHOTARD <patrice.chotard@foss.st.com>
---
 arch/arm/mach-stm32/Kconfig | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/arch/arm/mach-stm32/Kconfig b/arch/arm/mach-stm32/Kconfig
index de9d8547e61..1c6de559142 100644
--- a/arch/arm/mach-stm32/Kconfig
+++ b/arch/arm/mach-stm32/Kconfig
@@ -10,7 +10,6 @@ config STM32F4
 	select PINCTRL_STM32
 	select RAM
 	select STM32_RCC
-	select STM32_RESET
 	select STM32_SDRAM
 	select STM32_SERIAL
 	select STM32_TIMER
@@ -27,7 +26,6 @@ config STM32F7
 	select PINCTRL_STM32
 	select RAM
 	select STM32_RCC
-	select STM32_RESET
 	select STM32_SDRAM
 	select STM32_SERIAL
 	select STM32_TIMER
@@ -47,7 +45,6 @@ config STM32H7
 	select RAM
 	select REGMAP
 	select STM32_RCC
-	select STM32_RESET
 	select STM32_SDRAM
 	select STM32_SERIAL
 	select STM32_TIMER
-- 
2.47.2

